src/index.html

```diff
@@ -1,14 +1,27 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+<html class="timer-page" lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <p class="timer-page__title">
+      Hover over the stopwatch to stop it
+    </p>
+    <div class="stopwatch timer-page__stopwatch">
+
+      <div class="stopwatch__pointer"></div>
+      <div class="stopwatch__pointer"></div>
+      <div class="stopwatch__pointer"></div>
+      <div class="stopwatch__pointer"></div>
+
+      <div class="stopwatch__center"></div>
+      <div class="stopwatch__minutes"></div>
+      <div class="stopwatch__seconds"></div>
+    </div>
```

Comment:
If we say about the stopwatch as a separate element, I think we need to use the tag `section `ch p1. But I can be wrong.